cmd/controller-manager/app/options/options.go

```diff
@@ -63,8 +63,8 @@ func (s *ControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&s.Port, "port", s.Port, "The port that the controller-manager's http service runs on")
 	fs.StringVar(&s.ContentType, "api-content-type", s.ContentType, "Content type of requests sent to API servers")
 	fs.StringVar(&s.K8sAPIServerURL, "k8s-api-server-url", "", "The URL for the k8s API server")
-	fs.StringVar(&s.K8sKubeconfigPath, "k8s-kubeconfig", s.K8sKubeconfigPath, "Path to k8s core kubeconfig")
+	fs.StringVar(&s.K8sKubeconfigPath, "k8s-kubeconfig", "", "Path to k8s core kubeconfig")
 	fs.StringVar(&s.ServiceCatalogAPIServerURL, "service-catalog-api-server-url", "", "The URL for the service-catalog API server")
-	fs.StringVar(&s.ServiceCatalogKubeconfigPath, "service-catalog-kubeconfig", s.ServiceCatalogKubeconfigPath, "Path to service-catalog kubeconfig")
+	fs.StringVar(&s.ServiceCatalogKubeconfigPath, "service-catalog-kubeconfig", "", "Path to service-catalog kubeconfig")
```

Comment:
IIRC, if we have a default here, we'll never fall back to InClusterConfig, which is generally what we want to to.  By setting the default to empty, we effectively say "unless I'm told otherwise, use InClusterConfig"